docs/src/pages/guides/deploy.md

```diff
@@ -45,6 +45,9 @@ By default, the build output will be placed at `dist/`. You may deploy this `dis
 
    # navigate into the build output directory
    cd dist
+   
+   # add .nojekyll to bypass GitHub Page's default behavior
```

Comment:
I like this, can you then remove step 2 in the list above since its no longer needed?